TripKitDomainLegacy/build.gradle

```diff
@@ -13,7 +13,7 @@ android {
   buildToolsVersion versions.buildToolsVersion
 
   defaultConfig {
-    minSdkVersion 10
+    minSdkVersion 15
```

Comment:
Thanks for this change 👍 Should it be [`versions.proMinSdkVersion`](https://github.com/skedgo/tripkit-android/blob/dev/dependencies.gradle#L10) too?